.github/workflows/main.yml

```diff
@@ -28,18 +28,25 @@ jobs:
         with:
           path: gradle-build-plugins
 
+      # Setup Gradle properties, heap requirements are for the "Integration test w/ Nessie".
+      - name: Setup gradle.properties
+        shell: bash
+        run: |
+          mkdir -p ~/.gradle
+          echo "org.gradle.jvmargs=-Xmx1024m -XX:MaxMetaspaceSize=768m -Dfile.encoding=UTF-8" >> ~/.gradle/gradle.properties
+          echo "org.gradle.vfs.watch=false" >> ~/.gradle/gradle.properties
+
       - name: Set up JDK ${{ matrix.java-version }}
         uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
           java-version: 11
 
       - name: Build with Gradle
-        # Java 8 and 16+ don't run spotless, because google-java-format requires Java 11+ (but doesn't run w/ Java 16)
         uses: gradle/gradle-build-action@v2
         with:
           build-root-directory: gradle-build-plugins
-          arguments: --no-watch-fs assemble check publishToMavenLocal
```

Comment:
Q: i thought `--no-watch-fs` helps since CI doesnt do incremental buidls?

Reply:
That's what `org.gradle.vfs.watch=false` above does.